### PR TITLE
add Run3Summer23

### PR DESCRIPTION
**Pull request by sihyunjeon**

### CODE REVIEW

File: Campaigns/Run3Summer23/Run3Summer23.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "chain": "chain_Run3Summer23wmLHEGS_flowRun3Summer23DRPremix_flowRun3Summer23MiniAODv4_flowRun3Summer23NanoAODv12",
```
Comment (sihyunjeon) [inline Campaigns/Run3Summer23/Run3Summer23.json:2]: Nothing really changed except this

File: Campaigns/Run3Summer23/Run3Summer23.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+    },
+    "fragment_vars": {
+        "concurrentLHEforMadGraph5_aMCatNLO": "False",
+        "concurrentLHEforPowheg": "True",
+        "concurrentPS": "Concurrent"
+    },
+    "tune": "CP5",
+    "beam": 6800,
+    "gridpack_directory": "/eos/cms/store/group/phys_generator/cvmfs/gridpacks/PdmV/Run3Summer23"
```
Comment (sihyunjeon) [inline Campaigns/Run3Summer23/Run3Summer23.json:14]: and this, created the dir as well

**Comment by sihyunjeon** [comment]
@agrohsje @Cvico

**Comment by sihyunjeon** [comment]
please merge if one of you take a look

**Comment by sihyunjeon** [comment]
Also please check if you have a chance whether gridpack machinery gets 23 campaign select-able

**Comment by Cvico** [comment]
I've checked this works from GridpackMachine with a dummy test, the config file for 23 is fetched properly.

**Pull request merged**